Controller/MediaAdminController.php

```diff
@@ -42,22 +41,20 @@ public function createAction(Request $request = null)
     /**
      * {@inheritdoc}
      */
-    public function render($view, array $parameters = array(), Response $response = null, Request $request = null)
+    public function render($view, array $parameters = array(), Response $response = null)
```

Comment:
ping @core23 , please don't let this rot. @sonata-project/contributors , if you can chime in…